pkgs/by-name/mo/modrinth-app/package.nix

```diff
@@ -33,38 +34,55 @@ symlinkJoin rec {
     glib-networking
   ];
 
-  nativeBuildInputs = [
-    wrapGAppsHook3
-  ];
+  nativeBuildInputs = [ wrapGAppsHook3 ];
+
+  runtimeDependencies = lib.optionalString stdenv.isLinux (
+    lib.makeLibraryPath [
+      addDriverRunpath.driverLink
+
+      # glfw
+      libGL
+      xorg.libX11
+      xorg.libXcursor
+      xorg.libXext
+      xorg.libXrandr
+      xorg.libXxf86vm
+
+      # lwjgl
+      stdenv.cc.cc.lib
 
-  runtimeDependencies = lib.optionalString stdenv.isLinux (lib.makeLibraryPath [
-    addDriverRunpath.driverLink
-    flite # narrator support
+      flite # narrator support
 
-    udev # oshi
+      # openal
+      libpulseaudio
 
-    # lwjgl
-    libGL
-    libpulseaudio
-    stdenv.cc.cc.lib
-    xorg.libX11
-    xorg.libXcursor
-    xorg.libXext
-    xorg.libXxf86vm
-    xorg.libXrandr
-  ]);
+      udev # oshi
```

Comment:
Please place the comment at the same place, so either above or behind the library.